src/pepr/istio/index.ts

```diff
@@ -15,6 +20,8 @@ When(a.Pod)
   .WithLabel("batch.kubernetes.io/job-name")
   .WithLabel("service.istio.io/canonical-name")
   .Watch(async pod => {
+    Log.info(pod, `Processing Pod ${pod.metadata?.namespace}/${pod.metadata?.name}`);
```

Comment:
```suggestion
    Log.info(pod, `Processing Pod ${pod.metadata?.namespace}/${pod.metadata?.name} for istio job termination`);
```
Maybe? For clarity around where/for what this is getting processed.